src/actions/arcActions.ts

```diff
@@ -116,20 +116,20 @@ export async function getDAOData(avatarAddress: string, getDetails: boolean = fa
   const daoData: IDaoState = {
     avatarAddress,
     controllerAddress: "",
-    ethCount: Util.fromWei(await getBalance(avatarAddress)).toNumber(),
-    genCount: Util.fromWei((await votingMachineInstance.getTokenBalances({avatarAddress})).stakingTokenBalance).toNumber(),
+    ethCount: Number(Util.fromWei(await getBalance(avatarAddress))),
```

Comment:
really still seems to me like we should just put the case in fromWei for now...

Reply:
https://github.com/daostack/alchemy/pull/121/commits/45e2bce7b100057a537367d55738a3bd8c19f205